src/Styles/index.css

```diff
@@ -0,0 +1,92 @@
+@import "./star.css";
+@import "./button.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  max-width: 200px;
+  padding: 15px 16px;
+
+  font-family: "Roboto", sans-serif;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.card__image-link {
+  display: block;
+  margin: 17px 3px 0;
+}
+
+.card__image {
+  display: block;
+  width: 100%;
+  height: 134px;
+  text-align: center;
+}
+
+.card__info-name {
+  margin: 40px 0 0 ;
+
+  text-align: left;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__info-code {
+  margin: 4px 0 16px;
+
+  text-align: left;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__review-link {
+  text-decoration: none;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin: 24px 0 16px;
+}
+
+.card__price-name {
+  margin: 0;
+
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-tag {
+  margin: 0;
```

Comment:
That's better to reset margins (or anything else) using resetting styles, not here